utils.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
```

Comment:
This workflow only verifies that template is deployable in multiple regions. Seems like this file belongs to separate workflow.

Reply:
done